### ISSUE TITLE
Fedora removing WebKit1 library

### ISSUE THREAD

**Issue by mathstuf** [uzbl/uzbl#305]
https://bugzilla.redhat.com/show_bug.cgi?id=1375843

**Comment by mathstuf** [comment]
@keis We now have a deadline for Fedora at least.

**Comment by keis** [comment]
ah. nothing like a deadline :)

**Comment by colinbooth** [comment]
Debian too:
[https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=790216](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=790216)

**Comment by keis** [comment]
Thanks for the heads up @colinbooth 

Any idea what the timeline for debian is?

**Comment by colinbooth** [comment]
Not sure, the [current timeline](https://wiki.debian.org/DebianStretch) says full freeze goes in effect 2017-02-05. Going by the Jessie release history, you get about five months from that point before it ships. Also, you might want to contact the maintainer, there are several outstanding bugs and the version currently available is the may 2012 release.

**Comment by lordheavy** [comment]
And now a similar issue with ArchLinux:
https://lists.archlinux.org/pipermail/arch-dev-public/2017-January/028634.html

**Comment by GSI** [comment]
To clarify: **webkitgtk2** currently refers to v2.4, while **webkit2gtk** refers to 2.14. From Arch repos:

    extra/webkitgtk2 2.4.11-3
        GTK+ Web content engine library for GTK2

    extra/webkit2gtk 2.14.3-1
        GTK+ Web content engine library

**Comment by GSI** [comment]
For anybody interested in sandboxing uzbl, I just published an according **firejail** profile:

https://github.com/netblue30/firejail/pull/1060

**Comment by keis** [comment]
The version confusion in webkit is fantastic, eh?

The porting is towards the webkit2 api. I'm not exactly sure when this was introduced but webkit1 was dropped in webkitgk+ 2.6 IIRC. If it's 2.10 or 2.14 should not matter much.

**Comment by keis** [comment]
Heard through the grapevine (IRC) that gentoo have dropped webkit1

**Comment by mathstuf** [comment]
Yeah, Fedora has. I've been getting broken dep emails for a few weeks >.> . Now that scrolling is there, are we close enough for a "pre-release" of 1.0?

**Comment by keis** [comment]
I don't have a full grasp on what's missing but one big one in sharing cookies (#230), but at this pace it looks like the webkit1 dependency is quickly making 0.9 obsolete whether we like it or not.
